bootstrap/bootstrap.php

```diff
@@ -1,2 +1,23 @@
 <?php
-require_once '..\vendor\autoload.php';
\ No newline at end of file
+require_once '..\vendor\autoload.php';
+require_once 'database_config.php';
+
+$loader = new Twig_Loader_Filesystem('..\html');
+$twig = new Twig_Environment($loader);
+
+$conn = \database\DbConnection::getInstance();
+$base = null;
+$crud = null;
+if (DATABASE_IN_USE == 'mysql') {
+	$base = $conn->connectMySql();
+	$crud = new \database\CrudDatabaseMySql($base, new \src\Student());
```

Comment:
you keep insisting in passing Student instance to CrudDatabaseMySql counstructor, but you still keep instantiating the same object in that class... kinda defeats the purpose, don't you think? :) 
try renaming the "CrudDatabaseMySql" class to something like "StudentServiceMySql" - hopefully that hint will get you back to right track